srv/github.go

```diff
@@ -54,7 +53,18 @@ func NewGitHub(apiKey, org string) GitHub {
 	return &gitHub{apiKey, org, client}
 }
 
-func (g *gitHub) Releases(project string, all bool) ([]*Release, error) {
+func (g *gitHub) Latest(project string) (*Release, error) {
+	release, resp, err := g.client.Repositories.GetLatestRelease(context.Background(), g.org, project)
+	if r := newRelease(release); r == nil || resp.StatusCode == http.StatusNotFound {
+		return nil, ErrNotFound
+	} else if err != nil {
```

Comment:
Nope, if resp.StatusCode is 404, for the API is an error, but not for us. That's why the other check is first.